src/Model/Application.ts

```diff
@@ -0,0 +1,22 @@
+import { Entity, PrimaryGeneratedColumn, ManyToOne, ManyToMany, JoinTable, BaseEntity } from 'typeorm';
+import { Participant } from './Participant';
+import { Course } from './Course';
+import { Company } from './Company';
+
+@Entity()
+export class Application extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  application_id!: number;
```

Comment:
Since the class name is already Application, the primary key can be just id, no need to specify it as application_id. If it causes many changes, just leave it as it is, but a note for the future.